hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/metrics2/util/SampleQuantiles.java

```diff
@@ -243,7 +245,12 @@ synchronized public Map<Quantile, Long> snapshot() {
     
     Map<Quantile, Long> values = new TreeMap<Quantile, Long>();
     for (int i = 0; i < quantiles.length; i++) {
-      values.put(quantiles[i], query(quantiles[i].quantile));
+      /* eg : effectiveQuantile for 0.99 with inverseQuantiles will be 0.01. 
```

Comment:
Leaning into OOO: I might make an inversequantile class that overrides this function to keep it simple rather than overloading quantile with multiple definitions.

Reply:
updated